core/server/web/admin/app.js

```diff
@@ -1,30 +1,30 @@
-const debug = require('ghost-ignition').debug('admin'),
-    express = require('express'),
-
-    // App requires
-    config = require('../../config'),
-    constants = require('../../lib/constants'),
-    urlService = require('../../services/url'),
-
-    // Middleware
-    // Admin only middleware
-    adminMiddleware = require('./middleware'),
-    serveStatic = require('express').static,
-
-    // Global/shared middleware
-    cacheControl = require('../middleware/cache-control'),
-    urlRedirects = require('../middleware/url-redirects'),
-    errorHandler = require('../middleware/error-handler'),
-    maintenance = require('../middleware/maintenance'),
-    prettyURLs = require('../middleware/pretty-urls');
+const debug = require('ghost-ignition').debug('admin');
+const express = require('express');
+
+// App requires
+const config = require('../../config');
+const constants = require('../../lib/constants');
+const urlService = require('../../services/url');
+
+// Middleware
+// Admin only middleware
+const adminMiddleware = require('./middleware');
+const serveStatic = require('express').static;
+
+// Global/shared middleware
+const cacheControl = require('../middleware/cache-control');
+const urlRedirects = require('../middleware/url-redirects');
+const errorHandler = require('../middleware/error-handler');
+const maintenance = require('../middleware/maintenance');
+const prettyURLs = require('../middleware/pretty-urls');
 
 module.exports = function setupAdminApp() {
     debug('Admin setup start');
     const adminApp = express();
 
     // First determine whether we're serving admin or theme content
     // @TODO finish refactoring this away.
-    adminApp.use(function setIsAdmin(req, res, next) {
+    adminApp.use((req, res, next) => {
```

Comment:
Not sure we want to use arrow functions here 🤔 This is registered as named function in the express stack.

Reply:
This specific middleware will be removed by the other [PR](https://github.com/TryGhost/Ghost/pull/9884) but it makes sence to not rename it. Reverting :smiley:

Reply:
> This specific middleware will be removed

Yeah you are right :) But i think it's a nice example where we should not use arrow functions.